### DISCUSSION TITLE
Can't use floatingUI offset

### DISCUSSION THREAD

**Discussion by Gabsato02** [shipshapecode/shepherd#2411] [Q&A]
Hey guys, I need some help here. I'm trying to use floatingUI offset, but can't because the floatingUIOptions.middleware option expects a method.

`
tour.addStep({
...,
floatingUIOptions: {
    middleware: [offset({ mainAxis: 0, crossAxis: 12 })]
}
})
`
`// offset is not defined`

Documentation is not so clear in this regard. Where should I be importing this method from?

**Comment by AdrianCavazos** [comment]
I don't know how floatingUI works, but instead I use popper:
popperOptions: {
    modifiers: [{name: 'offset', options: {offset: [0, 24]}}]
}
